src/main/java/com/mongodb/kafka/connect/sink/MongoSinkTopicConfig.java

```diff
@@ -597,7 +596,6 @@ public NamespaceMapper getNamespaceMapper() {
     String prefix = format("%s%s.", TOPIC_OVERRIDE_PREFIX, topic);
     List<String> topicOverrides =
         props.keySet().stream().filter(k -> k.startsWith(prefix)).collect(Collectors.toList());
-    logObsoleteProperties(topicOverrides);
```

Comment:
I realized that `MongoSinkTopicConfig.validateAll` is called from `ConfigDef.validateAll` (see `MongoSinkConfig.createConfigDef`) with the same `props` as those passed to `ConfigDef.validateAll`. I.e., this call results in duplicate logging.